tests/integration/targets/iso/tasks/main.yml

```diff
@@ -46,7 +46,7 @@
       register: result
     - ansible.builtin.assert:
         that:
-          - "{{result.records.0.ready_for_insert}} is true"
+          - result.records.0.ready_for_insert is true
```

Comment:
Good catch

Reply:
A linter, not me :) but thanks anyway :D